packages/manager/modules/product-offers/src/product-offers.html

```diff
@@ -120,6 +121,21 @@
             >
             </ovh-payment-method-choice>
         </div>
+        <div
+            data-ng-if="$ctrl.workflow.isFreePricing() && $ctrl.workflow.hasComsuptionOption()"
+            class="row"
+        >
+            <oui-message type="info" class="mb-3">
+                <span
+                    class="oui-message__icon oui-icon oui-icon-info-circle"
+                    aria-hidden="true"
+                ></span>
+                <p
+                    class="oui-message__body"
+                    data-translate="ovh_product_offer_comsuption_alert"
+                ></p>
+            </oui-message>
```

Comment:
What about transcluding this part ? If we later have different messages to display ?

Reply:
this message apply for all pay as you go, isn't needed to transclude it i think